standalone/ansible/roles/postgres_replication/tasks/master.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Master | create replication user account
```

Comment:
To avoid weird connotations, could we change the terminology (names and filename) to `Primary`?